projects/1.16/assets/reauth/reauth/lang/zh_cn.json

```
@@ -1,14 +1,68 @@
 {
   "reauth.open": "开启ReAuth",
-  "reauth.gui.auth.title": "重新验证屏幕",
+  "reauth.gui.auth.title": "重新身份验证屏幕",
+  "reauth.gui.title.main": "ReAuth",
+  "reauth.gui.title.flow": "",
+  "reauth.gui.title.offline": "选择一个用户名",
   "reauth.gui.auth.username": "用户名",
   "reauth.gui.auth.update": "§9可用更新：%s",
   "reauth.gui.button": "ReAuth登录",
+  "reauth.gui.button.save": "将配置保存到配置文件",
+  "reauth.gui.button.save.tooltip": "警告：任何有权访问您的配置文件的人都可以提取和解密存储的配置。",
+  "reauth.gui.close": "X",
+  "reauth.gui.noProfile": "使用保存的配置登录",
+  "reauth.gui.profile": "作为%s登录",
+  "reauth.gui.text.profile": "使用保存的配置登录",
+  "reauth.gui.text.microsoft": "使用Xbox账户登录",
+  "reauth.gui.text.offline": "选择一个离线用户名",
+  "reauth.gui.button.authcode": "此设备",
+  "reauth.gui.button.devicecode": "所有设备",
+  "reauth.gui.button.offline": "选择用户名",
+  "reauth.gui.button.username": "设置用户名",
   "reauth.status.valid": "在线：§l§a✔",
-  "reauth.status.invalid": "在线: §l§c✘",
-  "reauth.status.unknown": "在线: §l§7?",
+  "reauth.status.invalid": "在线：§l§c✘",
+  "reauth.status.unknown": "在线：§l§7?",
+  "reauth.status.refreshing": "在线：§l§b?",
+  "reauth.status.error": "在线：§l§c?",
   "reauth.login.success": "§a登录成功！",
-  "reauth.login.fail": "§4登录失败: %s",
-  "reauth.login.error": "§4错误: %s",
-  "reauth.retry": "以%s身份登录并重试"
+  "reauth.login.fail": "§4登录失败：%s",
+  "reauth.login.error": "§4错误：%s",
+  "reauth.retry": "作为%s登录并重试",
+  "reauth.retry.disabled": "使用保存的配置登录并重试",
+  "reauth.msauth.code.success": "与Microsoft的身份验证成功",
+  "reauth.msauth.code.success.close": "你现在可以关闭此页面",
+  "reauth.msauth.code.fail.cancelled": "根据你的请求，取消了与Microsoft的身份验证",
+  "reauth.msauth.code.fail.server": "与Microsoft的身份验证暂时不可用",
+  "reauth.msauth.code.fail.unknown": "与Microsoft的身份验证失败",
+  "reauth.msauth.code.retry": "请单击此处重试",
+  "reauth.msauth.code.error.http200": "错误：这不是错误！？",
+  "reauth.msauth.code.error.http400": "错误：身份验证失败",
+  "reauth.msauth.code.error.http404": "错误：此资源不存在",
+  "reauth.msauth.code.error.http405": "错误：请求方法不正确",
+  "reauth.msauth.code.error.http415": "错误：无效的身份验证数据格式",
+  "reauth.msauth.code.error.http501": "错误：未知的请求方法",
+  "reauth.msauth.step.initial": "正在启动身份验证进程",
+  "reauth.msauth.step.finished": "登录成功",
+  "reauth.msauth.step.profile": "正在保存配置",
+  "reauth.msauth.step.failed": "登录失败",
+  "reauth.msauth.step.crypto": "正在初始化配置加密",
+  "reauth.msauth.step.yggdrasil": "使用Mojang进行身份验证",
+  "reauth.msauth.step.microsoft.code.await": "请查看浏览器进行登录",
+  "reauth.msauth.step.microsoft.code.redeem": "使用Microsoft进行身份验证",
+  "reauth.msauth.step.microsoft.device.request": "使用Microsoft设置身份验证",
+  "reauth.msauth.step.microsoft.device.poll": "请输入密码\n$%2$s\n在%1$s",

```

原文是`code` 记得修改1.18和1.19的
```suggestion
  "reauth.msauth.step.microsoft.device.poll": "请输入代码\n$%2$s\n在%1$s",
```